api/src/services/graphql/subscription.ts

```diff
@@ -18,9 +18,9 @@ export function bindPubSub() {
 	});
 }
 
-export function createSubscriptionGenerator(gql: GraphQLService, event: string) {
+export async function createSubscriptionGenerator(gql: GraphQLService, event: string) {
```

Comment:
As above, I believe this is redundant as the await is within the inner returned function

```suggestion
export function createSubscriptionGenerator(gql: GraphQLService, event: string) {
```